### DISCUSSION TITLE
[BUG] NPCs disapearing

### DISCUSSION THREAD

**Discussion by chadmoriarty007** [ErinaSugino/Starbound-SBR-Mod#129] [Trouble Shooting] [answered]
every time I strike an NPC with the Aphrodite´s Bow the NPC dissapears, I can still interact with them but my character dissapears too and no POV displays
[starbound.log](https://github.com/user-attachments/files/16616027/starbound.log)
![1](https://github.com/user-attachments/assets/ae38459c-8c94-40e4-b58d-f2711badec0e)

I´m sending also a list of my current mods... I followed very step of the installing guide but still is not working... sorry for the troubles I´m causing you

**Comment by ErinaSugino** [comment]
The issue is that your Sexbound installation is trying to load an old file "event.lua" which doesn't exist, preventing sexnode functionality from ever loading. I have seen this particular issue floating around a couple times by now, have yet to find the exact cuplrit of it though. This stuff has been removed a few SBR version ago, as it is a fully non-functional leftover from old SxB that serves no purpose, other than enabling users to create giant quantities of log spam for no reason. The only way this issue can come up is by having a mod that patches or overrides the core files of Sexbound with old, or faulty new code. Since the error crashes everything within the first couple lines of loaded code, however, I cannot even say which case it is.

Seeing as you have quite a few mods installed, asking to remove them all then readd them one by one until the issue reappears is a tall ask. However, I cannot provide you with more insight than that, cause we, too, are still trying to figure out who is breaking stuff here.

Though I do gotta wonder, considering there have been reports of things working just by "redownloading Sexbound" in the past and the fact that you seem to possess an unofficial, non-existent *future version* of POV Redux - where did you download SBR and POV Redux from?

**Comment by chadmoriarty007** [comment]
Hi! I´ve downloaded SBR from the loverslab.com site and the "future" version of POV Redux can be explained :) every time an update of any POV launches I unpack the .pak file of POV Redux and update the sprites and pack the files again and for my own account I rename the file, that´s why I´m owning a future version. I only update the sprites not any files, same goes to the "get rid of that underwear" addon as well... hope those "updates don´t mess with SBR. If you manage to find the problem it will be a bless because you managed to make excellent mods and I would really love to try SBR. Once again thanks for the reply and the help

**Reply by ErinaSugino** [reply]
Well, that rules out that current theory...
So I'm afraid in that case we (which includes you) have a bit of work ahead of us.

First I need you to backup your savefile - just copy the entire "storage" folder somewhere for savekeeping.
Then you need to remove (temporarily) any mod that isn't SBR and wipe the universe - cause you have mods like FU which WILL explode your savefile when uninstalled. From there on you just load into a fresh character on a fresh universe, use /admin and /spawnitem woodtable to place down a sexnode, and check the log file if it shows the same event.lua error. I would assume, and certainly hope, it doesn't.

Assuming everything works fine now, you'll have to re-add all your old mods one by one until the error reappears. And hopefully we find the culprit.

**Comment by chadmoriarty007** [comment]
Ah! ok, I´ll start right now and let´s see who is the one... Thanks

**Reply by ErinaSugino** [reply, marked as answer]
Update. Either POV Redux or Teratophilia Scenes is the issue. If you have the wrong, old base version of either of those, that will cause the error. Since you have a custom named file for TeratoScenes and a manually tampered with version of Redux, can't tell you which one. Problebly TeratoScenes. Just download the right fresh version (The one starting with "SxB_") and it should be fine.

**Reply by Wazki12** [reply]
Hi love your work it was the terato scenes the script folder inside specifically at least for me.

**Comment by DrTenebris** [comment]
![image](https://github.com/user-attachments/assets/0500d982-0f19-41ed-9104-24d48a60609a)
I am experiencing what I think is this same issue with the exception that I have double, triple, and quadruple checked that those mods are up to date for me. this is not my complete modlist, I use a combination of manual and workshop mods so most of my mods are on workshop, ill send that too... though I might fall into the "what horrendous things you have done to Starbound" category...

on account of my workshop mod count being 209.

**Reply by DrTenebris** [reply]
removed irrelevant modlist

**Reply by DrTenebris** [reply]
update, ive narrowed it to something in the manual mods folder, thank god I don't have to go through 209 mods trying to find what's breaking it.

**Reply by DrTenebris** [reply]
I found it, and in hindsight it was so hilariously obvious.

was futanari

**Reply by DrTenebris** [reply]
also deleting my workshop modlist from here because its unnecessary now
